src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1386,6 +1388,17 @@ internal override IEnumerable<FieldSymbol> GetFieldsToEmit()
                             yield return associatedField;
                         }
                         break;
+                    case SymbolKind.Property:
+                        if (m is SourcePropertySymbol propertySymbol)
+                        {
+                            FieldSymbol? backingField = propertySymbol.BackingField;
```

Comment:
>BackingField [](http://example.com/codeflow?start=71&length=12)

Given the proposal to revert ```BackingField```s behavior, we will need to use a different API here. A new API that has a purpose to bind accessors if necessary.